sabre1041/managing-ocp-install-beyond
# WIP - Lab 6

Guide for Lab to create and Ansible Playbook Bundle and scale the cluster

To Test:

1. Pull down changes included in this branch and deploy a brand new lab environment
2. Open Ansible Tower UI in web browser
    1. Go to _Projects_ tab and edit _Managing OCP from Install and Beyond_ project
    2. Change scm/branch to `lab6`
    3. Make sure `Clean` and `Delete on Update` are checked
    4. Click **Save**
    5. Trigger update of  _Managing OCP from Install and Beyond_ project
3. Run `0-Self-Configure` job template to configure tower
4. Run `1-Deploy_OpenShift_on_AWS` job template to deploy OpenShift
5. A new user has been created as part of this PR called `<student_id>-admin`. The password for this account is the same as the `<student_id>` appended by `admin` resulting in `<student_id_password>admin`
6. Grant `cluster-admin` access to the `<student_id_password>admin` user
    ```
    oc adm policy add-cluster-role-to-user cluster-admin `<student_id_password>admin`
    ```
7. Run through the steps in the lab guide
